components/release/development/release_service_config.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: ReleaseServiceConfig
+metadata:
+  name: release-service-config
```

Comment:
I am fine with either name, but it should be the same across envs

Reply:
either name? This is the name that the controller expects

Reply:
I see the issue. thanks